src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
```

Comment:
Consider moving font-face declarations to a separate file for better organization and maintainability. 📁